site/en/docs/extensions/migrating/mv2-sunset/index.md

```diff
@@ -16,7 +16,7 @@ tags:
 
 {% endAside %}
 
-As Manifest V3 approaches full feature parity with Manifest V2, we'll be progressively phasing out Manifest V2. This page specifies the timetable for this deprecation and describes the meaning of each milestone.
+As Manifest V3 approaches full feature parity with Manifest V2, we'll be progressively phasing out Manifest V2. See [Migrating to Manifest V3](https://developer.chrome.com/docs/extensions/migrating/) for instructions on how to convert your extension. This page specifies the timetable for this deprecation and describes the meaning of each milestone.
```

Comment:
I might read 'This page' as what you just linked to instead of the page I'm on. swapping the last two sentences would fix this.

Reply:
Good one 👍 I changed it. PTAL